lib/admin/appdata/cleanup.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+var mbaasRequest = require('../../mbaasRequest/mbaasRequest')
+  , constants = require('../../config/constants')
+  , config = require('../../config/config');
+
+/**
+ * Start a new tmp data cleanup job
+ * @param numNodes The number of nodes this cluster has
```

Comment:
Is `@param {Number} params.numNode - The number of nodes this cluster has`  better?